CONTRIBUTING.md

```diff
@@ -18,31 +18,34 @@
 
 Thus, this tool aims to provide a way to reproducibly deploy Dynatrace monitoring configuration in a "configuration as code"-way.
 
-As all things Dynatrace, scalability is an important requirement, both in number of configuration files and number of environments.
-This is also the area currently offering the most opportunity for improvement of the tool.
+As with all things Dynatrace, scalability is an important requirement, both in the number of configuration files and the number of environments.
+This is also the area currently offering the most opportunity to improve of the tool.
 
 Another way to contribute is to extend the list of supported APIs. Please take a look at 
 [the detailed instructions](https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/blob/main/docs/how-to-add-a-new-api.md).
 
 ## How to contribute
 
-The easiest way to start contributing or helping with the Monitoring as Code project is to pick an existing [issue/bug](#https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/issues) and [get to work](#building-the-Dynatrace-Monitoring-as-Code-Tool).
+The easiest way to start contributing or helping with the Monitoring as Code project is to pick an existing [issue/bug](https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/issues) and [get to work](#building-the-Dynatrace-Monitoring-as-Code-Tool).
 
-For proposing a change, we seek to discuss potential changes in GitHub issues in advance before implementation. That will allow us to give design feedback up front and set expectations about the scope of the change, and, for larger changes, how best to approach the work such that the Monitoring as Code team can review it and merge it along with other concurrent work. This allows to be respectful of the time of community contributors.
+For proposing a change, we seek to discuss potential changes in GitHub issues in advance before implementation. 
+That will allow us to give design feedback up front and set expectations about the scope of the change and, for more significant changes, 
+how best to approach the work such that the Monitoring as Code team can review it and merge it with other concurrent work. 
+This allows being respectful of the time of community contributors.
 
-The repo follows a rather standard branching & PR workflow.
+The repo follows a relatively standard branching & PR workflow.
 
 Branches naming follows the `feature/{Issue}/{description}` or `bugfix/{Issue}/{description}` pattern.
 
 Branches are rebased and only fast-forward merges to main permitted. No merge commits.
 
-Commits are not auto-squashed when merging a PR, so please make sure your commits are fit to go into main (DIY squash when necessary).
+Commits are not auto-squashed when merging a PR, so please ensure your commits are fit to go into main (DIY squash when necessary).
```

Comment:
technically incorrect now, we allow squashing as a merge strategy, maybe you could adapt this to: 

```suggestion
By default, commits are not auto-squashed when merging a PR, so please ensure your commits are fit to go into main. 

For convenience auto-squashing all PR commits into a single one is an optional merge strategy - but we strive for [atomic commits](https://www.freshconsulting.com/insights/blog/atomic-commits/) with [good commit messages](https://cbea.ms/git-commit/) in main so not auto-squashing is recommended.
```